src/main/kotlin/com/hedvig/lookupservice/visma/service/VismaService.kt

```diff
@@ -0,0 +1,67 @@
+package com.hedvig.lookupservice.visma.service
+
+import com.hedvig.lookupservice.util.toRealEstateEntity
+import com.hedvig.lookupservice.util.toRealEstateResponse
+import com.hedvig.lookupservice.visma.client.VismaFeignClient
+import com.hedvig.lookupservice.visma.repository.VismaRepository
+import com.hedvig.lookupservice.visma.web.dto.RealEstateResponse
+import com.hedvig.lookupservice.visma.web.exception.RealEstateException
+import com.hedvig.lookupservice.visma.web.exception.RealEstateSearchException
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import java.time.LocalDateTime
+
+@Service
+class VismaService @Autowired constructor(
+    val vismaFeignClient: VismaFeignClient,
+    val vismaRepository: VismaRepository
+) {
+
+    @Value("\${visma.days.to.use.cache:30}")
+    val daysToUseCache: Long = 30
```

Comment:
Default value ":30" already solves this

```suggestion
    val daysToUseCache: Long
```

Reply:
unfortunately i added this because kotlin will complain about it is not initialized